src/types/DrinkRecipe.ts

```diff
@@ -22,6 +22,7 @@ export interface MeasuredDrinkRecipeIngredient {
   unit: "oz" | "tsp" | "Drop" | "Dash" | "Tbsp" | "Barspoon" | "Cup"
   unitAmount: number
   prefix?: string
+  suffix?: string
```

Comment:
I left off suffix for measured and numbered ingredients on purpose. My reasoning for that was that a "suffix" actually changes the ingredient eg "Maple Syrup" -> "Maple Syrup Grade A". A prefix, e.g. "Chilled" is just something that you do to prep that ingredient. IMO, I think the ingredients with the "Grade X" suffixes should have those suffixes right in the ingredient name and then add `"Maple Syrup Grade A": ["Syrup"]` to the ingredient aliases list (and also for the grade b version). Then we can remove the suffix attribute from both these types again too. What do you think?